packages/tools/kadena-cli/src/utils/helpers.ts

```diff
@@ -329,3 +330,25 @@ export const maskStringPreservingStartAndEnd = (
     return `${startChars}${maskChar.repeat(maskCharLength)}${endChars}`;
   }
 };
+
+export const isNotEmptyString = (value: unknown): value is string =>
+  value !== null && value !== undefined && value !== '';
+
+/**
+ * Prints zod error issues in format
+ * ```code
+ * {key}: [issues by key]\n
+ * ...repeat
+ * ```
+ */
+export const formatZodError = (error: ZodError): string => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const format = error.format() as any;
+  const formatted = Object.keys(format)
+    .map((key) => {
+      if (key === '_errors') return null;
+      return `${key}: ${format[key]?._errors.join(', ')}`;
+    })
+    .filter(notEmpty);
+  return formatted.join('\n');
+};
```

Comment:
I copied these from this https://github.com/kadena-community/kadena.js/pull/1647 . Maybe after this branch merged i'll rebase it

Reply:
Nice!